plugin/apply/apply.go

```diff
@@ -47,10 +47,11 @@ type gen struct {
 }
 
 func (g *gen) Add(name string, typs []types.Type) (string, error) {
-	if _, _, err := g.parseTypes(name, typs); err != nil {
+	sig, _, err := g.parseTypes(name, typs)
+	if err != nil {
 		return "", err
 	}
-	return g.SetFuncName(name, typs...)
+	return g.SetFuncName(name, derive.RenameBlankIdentifier(sig), typs[1])
```

Comment:
Why not use last arg? as in
```go
sig, lastArg, err := g.parseTypes(name, typs)
...
return g.SetFuncName(name, derive.RenameBlankIdentifier(sig), lastArg)
```

Reply:
Yeah, I think we can use `lastArg` since `typs[1]` is used only for validation, not generating code.